src/hotspot/share/memory/metaspace/blockTree.hpp

```diff
@@ -212,30 +212,30 @@ class BlockTree: public CHeapObj<mtMetaspace> {
     } else {
       assert(child == _root, "must be root");
       _root = replace;
-      if (replace != NULL) {
-        replace->_parent = NULL;
+      if (replace != nullptr) {
+        replace->_parent = nullptr;
       }
     }
     return;
   }
 
   // Given a node n and an insertion point, insert n under insertion point.
   void insert(Node* insertion_point, Node* n) {
-    assert(n->_parent == NULL, "Sanity");
+    assert(n->_parent == nullptr, "Sanity");
     for (;;) {
       DEBUG_ONLY(check_node(insertion_point);)
       if (n->_word_size == insertion_point->_word_size) {
-        add_to_list(n, insertion_point); // parent stays NULL in this case.
+        add_to_list(n, insertion_point); // parent stays null in this case.
```

Comment:
Small nit. Do we have a unified naming scheme for comments?
 
"null" reads like a java null. I prefer keeping NULL in comments.

Reply:
FWIW, from fixing these reviews I've never seen us referring to a Java null when we meant a C++ nullptr.